Lib/Image2D/image2d.pd

```diff
@@ -386,9 +386,9 @@ sub PDL::conv2d {
    my $opt; $opt = pop @_ if ref($_[$#_]) eq \'HASH\';
    die \'Usage: conv2d( a(m,n), kern(p,q), [o]b(m,n), {Options} )\'
```

Comment:
Might be worth looking at the `die \'Usage:` statements in this file.

Reply:
The "die" statements are Ok as far as I can see, since they refer to the signatures, which haven't changed.